src/pages/Proposal.vue

```diff
@@ -1,4 +1,81 @@
 <template lang="pug">
-div
-    h1 Proposal
+q-page(padding)
+  div.row.justify-between.items-center.q-pt-lg.q-pb-sm
+    h1.text-h5.q-ma-none Multisig Transactions
+
+  q-tabs(v-model="tab" align="left" active-color="primary" content-class="text-grey-7"  no-caps)
+    q-tab(v-if="isAuthenticated" name="myProposal" label="My proposals")
+    q-tab(name="allProposal" label="All proposals")
+
+  q-separator
+
+  q-tab-panels(v-model="tab")
+    q-tab-panel(v-if="isAuthenticated" name="myProposal").q-pa-none
+      div.q-py-lg
+        ProposalTable(
+          title="Needs your signature"
+          type="needsYourSignature"
+          :account="account"
+        )
+        ProposalTable(
+          title="Proposals created"
+          type="proposalsCreated"
+          :account="account"
+        )
+
+    q-tab-panel(name="allProposal").q-pa-none
+      div.q-py-lg
+        ProposalTable(
+          title="Proposals"
+          type="allProposals"
+          :account="account"
+          :blockProducers="blockProducers"
+        )
 </template>
+
+<script lang="ts">
+import { defineComponent, ref, onMounted } from 'vue';
+import ProposalTable from 'src/components/ProposalTable.vue';
+import { api } from 'src/api';
+import { useAuthenticator } from 'src/composables/useAuthenticator';
+
+export default defineComponent({
+  name: 'Proposal',
+  components: {
+    ProposalTable
+  },
+  setup() {
+    const blockProducers = ref<string[]>([]);
+    const { account, isAuthenticated } = useAuthenticator();
+
+    const tab = ref<'myProposal' | 'allProposal'>('myProposal');
+
+    onMounted(() => {
+      if (!isAuthenticated.value) {
+        tab.value = 'allProposal';
+      }
+    });
+
+    onMounted(async () => {
```

Comment:
duplicate `onMounted()`, combine unless it's supposed to be a different lifecycle hook

Reply:
According to the [Vuejs3 docs](https://vuejs.org/guide/extras/composition-api-faq.html#more-flexible-code-organization) repeating these functions `onMounted` is a good practice to use the Composition API of Vuejs3 because in this component we have two logical concerns: the first is to handle the tabs, and the second is to load block producers.

Reply:
gotcha, wasn't sure if it was intentional to sep logic.